sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>This notebook focuses on training the XGBoost algorithm and downloading Debugger reports rather than an end-to-end solution of the Higgs Boson challenge, I recommend to change the title and content to:</p><p> </p><h1>Amazon SageMaker Debugger XGBoost training report for Higgs Boson Detection Challenge</h1><p> </p><p>This tutorial walks thorugh an example of training an XGBoost model using data from the <a href="http://opendata.cern.ch/record/328" target="_blank">2014 ATLAS Higgs Boson Machine Learning Challenge</a>. This example showcases some of the new features available in SageMaker Debugger, such as the deep profiling and the XGBoost training report.</p><p> </p><p>The <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-profiling-report.html" target="_blank">Debugger profiling report</a> displays hardware resource utilization metrics such as cpu, gpu, memory, and IO utilization. Debugger will help you identify any hardware bottlenecks and appropriately choose the right-sized instance for your training job.</p><p> </p><p>The <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-training-xgboost-report.html" target="_blank">Debugger XGBoost training report</a> will provide a comprehensive evaluation of your model's performance to help you fine-tune and improve your model.</p><p> </p><p>The <a href="https://github.com/aws/sagemaker-python-sdk" target="_blank">SageMaker Python SDK</a> makes it easy to train XGBoost models accessing other AWS services, such as Amazon EC2, Amazon ECR, and Amazon S3. For more information about training XGBoost models using SageMaker, see the <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html" target="_blank">XGBoost Algoritm Documentation</a> and the <a href="https://github.com/aws/sagemaker-python-sdk" target="_blank">sagemaker-python-sdk</a> repository.</p><p>For more information on XGBoost, visit the <a href="https://xgboost.readthedocs.io/en/latest/." target="_blank">XGBoost official website</a>.</p><h3>Table of contents</h3><ul><li><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/files/#setup" target="_blank">Setup and imports</a></li><li><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/files/#data" target="_blank">Get and prepare data</a></li><li><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/files/#estimator" target="_blank">Create the SageMaker XGBoost Estimator</a></li><li><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/files/#train" target="_blank">Train XGBoost Model</a></li><li><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/files/#reports" target="_blank">View post training reports</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done